whetstone/compiler-test/src/test/kotlin/com/freeletics/mad/whetstone/codegen/FileGeneratorTestCompose.kt

```diff
@@ -576,7 +594,11 @@ internal class FileGeneratorTestCompose {
 
               public val testClass: TestClass
 
-              public val testClass2: TestClass2
+              public val test: TestClass2
```

Comment:
This changed because we are now using the parameter name here instead of a name derived from the class.